ddtrace/internal/telemetry/data/application.py

```diff
@@ -0,0 +1,17 @@
+import platform
+
+from ....settings import _config as config  # noqa: E402
+from ....version import get_version
+
+
+class Application:
+    def __init__(self):
+        self.service_name = config.service or "unnamed_python_service"  # type: str
+        self.service_version = config.version or "x.x.x"  # type: str
+        self.env = config.env or "dev"  # type: str
+        self.language_name = "python"  # type: str
+        self.language_version = platform.python_version()  # type: str
+        self.tracer_version = get_version()  # type: str
+        self.patches = ""  # type: str
+        self.runtime_name = ""  # type: str
+        self.runtime_version = ""  # type: str
```

Comment:
```suggestion
        self.runtime_name = platform.python_implementation()  # type: str
        self.runtime_version = get_version()  # type: str
```

Or really, maybe `runtime_version` we can keep optional?

We also need to think about how to get this information from PyPy/Jython/etc if the APIs are the same or not.

Also this is where `runtime_version` might matter. For CPython it should always be the same as the version of Python you are running, but PyPy or Jython the version of Python it supports might not be the same version scheme they use for the runtime version.